### DISCUSSION TITLE
Thanks to D. M. Coles for fixing build scripts

### DISCUSSION THREAD

**Discussion by SamuraiCrow** [EEC-Developers/eec#22] [Announcements]
The author of E-VO, @dmcoles , has fixed the build scripts and instructions to EEC so now it is possible to use.  Many thanks!

**Comment by dmcoles** [comment]
Glad to be able to help. I probably don't have time to make much in the way of major contributions to EEC as I already have several projects to work on but I will do a bit when I can.

**Reply by SamuraiCrow** [reply]
Welcome to the team!  Don't worry about time.  Amiga is only a hobby nowadays anyway.
